mobile-widgets/qmlinterface.h

```diff
@@ -315,6 +319,6 @@ public slots:
 
 	void diveSummaryTextChanged(QStringList);
 private:
-	QMLInterface();
+	QMLInterface() {};
```

Comment:
Could we stop inlining constructors without reason, please?
And sorry for being pedantic: The semi-colon after a function-definition here is superfluous, by the way.

Reply:
The semicolon was my error (I participate in 2 other projects, where it is demanded), it is gone.

What is the technical reason for not inlining the constructor ?

Our code have plenty of examples (all of qPref f.x.). It is a very convenient way to declare the constructor private (avoid construction of objects) and at the same time tell it is empty.